test/gtest/ucp/test_ucp_tag_mt.cc

```diff
@@ -82,3 +82,179 @@ UCS_TEST_P(test_ucp_tag_mt, send_recv) {
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_tag_mt)
+
+
+class test_ucp_rma_mt : public test_ucp_tag_mt {
```

Comment:
this code should be added to new file "test_ucp_rma_mt.cc" - it does not belong to "test_ucp_**tag**_mt.cc"